x-pack/plugins/ml/public/application/routing/routes/new_job/wizard.tsx

```diff
@@ -45,7 +45,7 @@ const multiMetricBreadcrumbs = [
   ...baseBreadcrumbs,
   {
     text: i18n.translate('xpack.ml.jobsBreadcrumbs.multiMetricLabel', {
-      defaultMessage: 'Multi metric',
+      defaultMessage: 'Multi-metric',
```

Comment:
When testing how this breadcrumb looks, I noticed that a recent PR #64667 changed one of the breadcrumbs for the 'Create job' stage to 'Create jobs' (plural). I think this should always be singular 'Create job'. All of the wizards other than the data recognizer one only create one job, and I don't think the text of the breadcrumb should change for the one use case of using the data recognizer wizard for a module which has multiple jobs. Would you be able to add that edit to this PR please - `x-pack/plugins/ml/public/application/routing/breadcrumbs.ts` `CREATE_JOB_BREADCRUMB`.

![image](https://user-images.githubusercontent.com/7405507/81073456-93fe7700-8edf-11ea-867f-cdea9ae7b8b9.png)

Reply:
Yup, I've added that change.